<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMovieMakerControls</class>
 <widget class="QWidget" name="QmitkMovieMakerControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovieMaker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Window for stepping</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="cmbSelectedStepperWindow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel3_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Window for recording</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="cmbSelectedRecordingWindow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Recording options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Step</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QmitkSliderNavigatorWidget" name="slidAngle" native="true"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QToolButton" name="btnPlay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="toolTip">
           <string>Start stepper</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../../mitkBinaryCTK/MITK-superbuild/MITK-build/Modules/Bundles/org.mitk.gui.qt.moviemaker/org_mitk_gui_qt_moviemaker_cached.qrc">
            <normaloff>:/org.mitk.gui.qt.moviemaker/resources/play.xpm</normaloff>:/org.mitk.gui.qt.moviemaker/resources/play.xpm</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnPause">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="toolTip">
           <string>Pause stepper</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../../mitkBinaryCTK/MITK-superbuild/MITK-build/Modules/Bundles/org.mitk.gui.qt.moviemaker/org_mitk_gui_qt_moviemaker_cached.qrc">
            <normaloff>:/org.mitk.gui.qt.moviemaker/resources/pause.xpm</normaloff>:/org.mitk.gui.qt.moviemaker/resources/pause.xpm</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnStop">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="toolTip">
           <string>Stop stepper</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../../mitkBinaryCTK/MITK-superbuild/MITK-build/Modules/Bundles/org.mitk.gui.qt.moviemaker/org_mitk_gui_qt_moviemaker_cached.qrc">
            <normaloff>:/org.mitk.gui.qt.moviemaker/resources/stop.xpm</normaloff>:/org.mitk.gui.qt.moviemaker/resources/stop.xpm</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QToolButton" name="btnMovie">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Write Movie ...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="btnScreenshot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="toolTip">
         <string>Writes a screenshot of the selected window.</string>
        </property>
        <property name="text">
         <string>Write Screenshot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="m_HRScreenshot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="toolTip">
         <string>Writes a screenshot of the selected window.</string>
        </property>
        <property name="text">
         <string>Write high resolution 3D Screenshot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Playing options</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QWidget" name="buttonGroup1" native="true">
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="rbtnSpatial">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Spatial</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnTemporal">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Temporal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnCombined">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Combined</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="textLabel2_2">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>S/T Relation</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spatialTimeRelation">
             <property name="font">
              <font/>
             </property>
             <property name="minimum">
              <number>-10</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line2">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonGroup2" native="true">
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="rbtnForward">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Forward</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnBackward">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Backward</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnPingPong">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Ping-Pong</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="textLabel2">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>Cycle (sec)</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnDuration">
             <property name="font">
              <font/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSliderNavigatorWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSliderNavigatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rbtnForward</tabstop>
  <tabstop>rbtnBackward</tabstop>
  <tabstop>rbtnPingPong</tabstop>
  <tabstop>rbtnSpatial</tabstop>
  <tabstop>rbtnTemporal</tabstop>
  <tabstop>cmbSelectedStepperWindow</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../../../mitkBinaryCTK/MITK-superbuild/MITK-build/Modules/Bundles/org.mitk.gui.qt.moviemaker/org_mitk_gui_qt_moviemaker_cached.qrc"/>
 </resources>
 <connections/>
</ui>
